tests/kda/test_generated_kda_indexed_prefill_source.py

```
@@ -0,0 +1,582 @@
+    if torch.cuda.get_device_capability() not in {(10, 0), (10, 3)}:
+        pytest.skip("the generated source package targets SM100a and SM103a")
+    flash_kda = pytest.importorskip("flash_kda")
+    flash_kda_C = pytest.importorskip("flash_kda_C")
+    reference_root = Path(reference_root_value).resolve(strict=True)
+    expected_reference_commit = os.environ.get("FLASH_KDA_REFERENCE_COMMIT")
+    if expected_reference_commit is None:
+        pytest.skip("FLASH_KDA_REFERENCE_COMMIT must pin the reference commit")
+    assert len(expected_reference_commit) == 40 and set(
+        expected_reference_commit
+    ) <= set("0123456789abcdef")
+    reference_commit = subprocess.check_output(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)